services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -95,12 +95,13 @@ class ApplicationController(
     @Operation(
         summary = "Update an application",
         description =
-            "Updates an application. The application can be updated until it has started " +
-                "processing on Allu, i.e. it's still pending. If the application has been " +
-                "sent to Allu, it will be updated there as well. If an Allu-update is " +
-                "required, but it fails, the local version will not be updated. The " +
-                "pendingOnClient value can't be changed with this endpoint. Use POST " +
-                "/hakemukset/{id}/send-application for that."
+            """Updates an application.
```

Comment:
First sentence may be redundant as it is already mentioned in the summary. Same thing with sending the application description.

Reply:
Thank you for the suggestion. I changed the first sentences to describe the return value instead.